koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -427,13 +444,21 @@ def createContainer(self, src, target_info, arch, scratch, yum_repourls, branch,
             'scratch': scratch,
             'koji_task_id': self.id,
         }
+        if arch:
+            create_build_args['platform'] = arch
+            create_method = self.osbs().create_build
+        elif arches:
```

Comment:
Could put the 'arches' test before the 'arch' test to make it clear `create_orchestrator_build` is the preferred method and the old method is there only for compatibility.